packages/liveblocks-core/src/crdts/__tests__/LiveObject.test.ts

```diff
@@ -405,7 +407,56 @@ describe("LiveObject", () => {
   });
 
   describe("acknowledge mechanism", () => {
-    describe("should ignore incoming updates if current op has not been acknowledged", () => {
+    it(
+      "should not ignore history updates if the current op has not been acknowledged",
+      prepareDisconnectedStorageUpdateTest<{
+        items: LiveObject<{ b?: string; a?: string }>;
+      }>(
+        [
+          createSerializedObject("0:0", {}),
+          createSerializedObject("0:1", { a: "B" }, "0:0", "items"),
+        ],
+        async ({ assert, batch, root, machine }) => {
+          const items = root.get("items");
+          machine.pauseHistory();
```

Comment:
Here,`pauseHistory` and `resumeHistory` should have no impact because batch already group the two `set`.